### DISCUSSION TITLE
VF2PP subgraph monomorphism

### DISCUSSION THREAD

**Discussion by mdindoost** [networkx/networkx#7563] [Q&A] [answered]
Is there a function based on the VF2++ algorithm for finding subgraph monomorphism?

**Comment by dschult** [comment, marked as answer]
There is a PR that prepares the vf2++ code for supporting the subgraph monomorphism problem. But it has not seen any action for a while. Luckily time is planned for focusing on vf2++ as well as other iso/monomorphism algorithms later this Fall and Winter. So for the moment we don't support subgraph monomorphism using VF2++. But help is on the way in the next year.

**Reply by lazyhope** [reply]
Hi just came across this discussion because I was looking for the progress of vf2++ implementation in networkx. Any updates for now?

Best.

**Reply by dschult** [reply]
Thanks for this inquiry. We are in the same state with VF2++ subgraph monomorphism as a year ago. 
PR #6230 proposes code for implementing subgraph isomorphism. The good news is that I have been able to
focus on the isomorphism suite more in the last 3 months, so I really do expect to make progress on this feature
over the next 1-3 months. Fingers crossed. :)
